libs/gi/ui/src/components/team/TeamCard.tsx

```diff
@@ -496,7 +496,7 @@ function ArtifactCard({ artifactData }: { artifactData: ArtifactData }) {
   const { setNum = {}, mains = {} } = artifactData
   const { t } = useTranslation('statKey_gen')
   const processedSetNum = Object.entries(setNum).filter(
-    ([, num]) => num === 2 || num === 4
+    ([, num]) => num === 2 || num === 4 || num === 5
```

Comment:
I think I want to show that the user have specific 2p/4p effects active, instead of how many they have. in the cases where they have 3p or 5p, i still want to show 2p/4p.

```suggestion
        .filter(([, num]) => num > 1)
        .map(([set, num]) => [set, num < 4 ? 2 : 4])
```

Reply:
Added the changes